test/create-run-publish-app.sh

```diff
@@ -12,6 +13,10 @@ dotnet run
 dotnet publish -o publish/framework-dependent
 
 echo "Testing self-contained deployment"
+if [ "rel-1.0.0-preview2.1" == "${2}" ]; then
```

Comment:
It seems like the logic for checking the image version fits better in run-test.sh.  What do you think about passing the path to the NuGet.Config as an optional second argument to create-run-publish.sh, rather than passing the sdk image version?  run-test.sh would pass in the path if and only if `development_image_version` equals `rel-1.0.0-preview2.1`.